src/components/common/EventList.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div v-infinite-scroll="loadMore" infinite-scroll-distance="80">
+    <template v-for="group in groupedEvents">
+      <div :key="group[0].title">
+        <h3>{{ group[0].title }}</h3>
+        <template v-for="item in group">
+          <slot v-bind="item.event" />
+        </template>
+      </div>
+    </template>
+  </div>
+</template>
+
+<script>
+import groupBy from "lodash.groupby"
+import orderBy from "lodash.orderby"
+import moment from "moment"
+
+const categories = [
+  {
+    title: "Today",
+    matcher: (event) => {
+      return moment(event.timestamp).isSame(moment(), "day")
+    },
+  },
+  {
+    title: "Yesterday",
+    matcher: (event) => {
+      return moment(event.timestamp).isSame(moment().subtract(1, "days"), "day")
+    },
```

Comment:
Doesn't it make more sense categories like the ones we are already conceptually using throughout the whole app?

I mean, `Governance`, `Staking` and `Transfers`

Reply:
Event is the same thing than notification?

Reply:
this is the same code as before in the transaction list. I just reduced this to be any event that would should in such a segmented list. In the case of the transaction list each transaction is an event.

Reply:
"In the case of the transaction list each transaction is an event."

? what do we gain here with the event thing? Ah, because you want to reuse the same component for notifications?

Reply:
It's reused for notifications, same component